src/app/photographer/[handle]/layout.tsx

```
@@ -0,0 +1,57 @@
+  }
+
+  const title = `${photographer.name} — ${photographer.companyName || "Photographer"} in ${photographer.location || ""}`;
+  const description = photographer.bio
+    ? photographer.bio.slice(0, 160)
+    : `${photographer.name} is a photographer on ISO. View their portfolio, reviews, and availability.`;
+
+  const ogImageUrl = `https://myiso.app/api/og/photographer?handle=${encodeURIComponent(handle)}`;
+
+  return {
+    title,
+    description,
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'description' should be before 'title'.

(sort-keys-fix/sort-keys-fix)